src/Shared/Shared.csproj

```diff
@@ -17,6 +17,9 @@
     <PackageReference Include="Microsoft.Azure.DurableTask.Core" />
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" />
     <PackageReference Include="Microsoft.Extensions.Hosting" />
+    <!-- For some reason, this project resolves this at 6.0.0 instead of the centrally defined 8.0.6. 
+         Since 6.0.0 has known vulnerabilities, we override to 6.0.11 to avoid major version changes here. -->
+    <PackageReference Include="System.Text.Json" VersionOverride="6.0.11"/>
```

Comment:
Do we need `VersionOverride` or can we just mention the package name and rely on the central package management config to get the version?

Reply:
Removing VersionOverride, you get 8.0.6. This is a major version change from the previously resolved 6.0.0, but if we're not concerned about that, we can do it

Reply:
Done

Reply:
Done